doc/classes/SubViewportContainer.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="SubViewportContainer" inherits="Container" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Control for holding [SubViewport]s.
+		A container used for holding and displaying the contents of a [SubViewport].
 	</brief_description>
 	<description>
-		A [Container] node that holds a [SubViewport]. It uses the [SubViewport]'s size as minimum size, unless [member stretch] is enabled.
-		[b]Note:[/b] Changing a SubViewportContainer's [member Control.scale] will cause its contents to appear distorted. To change its visual size without causing distortion, adjust the node's margins instead (if it's not already in a container).
-		[b]Note:[/b] The SubViewportContainer forwards mouse-enter and mouse-exit notifications to its sub-viewports.
+		A container used for holding and displaying the contents of a [SubViewport]. It uses the [SubViewport]'s size as minimum size, unless [member stretch] is enabled.
```

Comment:
You can copy my description from #71227 (except the last note).
I think "holding" should be removed from the description, because it's misleading.

Reply:
Did an "inbetween" of this. I'm not sure if this is intended, but your description was stated as if multiple SubViewports can provide their content to the SubViewPortContainer to display.

Reply:
It's intended, SubViewportContainer will display all underlying viewports.

Reply:
Sounds good, but you follow up the first sentence there as if there's still a single subviewport. Idk I don't get it

Reply:
Maybe
`It uses the combined size of the [SubViewport]s as minimum size, unless [member stretch] is enabled.`